template/web/app/paymaster-bundler/api/paymasterProxy/route.test.ts

```diff
@@ -0,0 +1,77 @@
+import { NextRequest } from 'next/server';
+import { paymasterClient } from '../../../../../web/src/components/SmartWallets/PaymasterClient';
```

Comment:
PaymasterClient feels more an utils, something that should live under:
- https://github.com/coinbase/build-onchain-apps/tree/main/template/web/src/utils/paymasterClient.ts

and than accessed as `@/utils/paymasterClient`

Reply:
moved!